esphome/components/tem3200/tem3200.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/components/sensor/sensor.h"
+#include "esphome/components/i2c/i2c.h"
+
+namespace esphome {
+namespace tem3200 {
+
+/// This class implements support for the tem3200 pressure and temperature i2c sensors.
+class TEM3200Component : public PollingComponent, public i2c::I2CDevice {
+ public:
+  void set_temperature_sensor(sensor::Sensor *temperature_sensor) { temperature_sensor_ = temperature_sensor; }
```

Comment:
https://esphome.io/guides/contributing#codebase-standards
> All uses of class members and member functions should be prefixed with this-> to distinguish them from global functions in code review.

```suggestion
  void set_temperature_sensor(sensor::Sensor *temperature_sensor) { this->temperature_sensor_ = temperature_sensor; }
```